server/interfaces/auth.ts

```diff
@@ -6,3 +6,8 @@ export interface UserData {
   phone: number;
   role: string;
 }
+
+export interface userLoginAttrs {
```

Comment:
why not userLoginAttributes?

Reply:
Because of this 
![Screenshot 2023-07-01 162735](https://github.com/GSG-G13/Balanty/assets/87927920/949c5f2b-6c34-4a7f-9e56-367f2c6162fd)